src/main/java/com/napier/sem/App.java

```diff
@@ -130,7 +154,9 @@ public static void main(String[] args)
         CSVHelper.WriteReportListToCSV(capitalCityReportList, "capital_city_reports");
         System.out.println("Generating Population CSV reports...");
         CSVHelper.WriteReportListToCSV(populationReportList, "population_reports");
-
+       //code by viva
+	    System.out.println("Generating Language CSV reports...");
```

Comment:
Remove space in front of System.out.println("Generating Language CSV reports...");

Reply:
Removed and updated.